cli/dockerutils.py

```diff
@@ -22,12 +22,17 @@ def chdir(dirname):
 
 class DockerUtils:
 
-    def docker_build(self, image_tag, docker_file):
+    def docker_build(self, image_tag, docker_file, build_args):
+        build_arg_str = ""
+        if build_args:
```

Comment:
Ah. I see. I forgot that python initializes missed args to a function call as nil.